assignments/arrays.js

```diff
@@ -63,34 +75,58 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
-
+for (let i = 0; i < inventory.length; i++) {
+  if (inventory[i].id === 33) {
+    let car = inventory[i];
+    console.log(`Car 33 is a ${car.car_year} ${car.car_make} ${car.car_model}`);
```

Comment:
I don't think we needed to loop here. Could have made it even simpler by just saying ```console.log('Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model'). I do like that you used the template literals though despite that they were not covered in the lecture